substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/image/NativeImageDebugInfoProvider.java

```diff
@@ -1197,6 +1266,27 @@ private void computeFullFilePath() {
             }
         }
 
+        @Override
+        public boolean isConstructor() {
+            return method.isConstructor();
+        }
+
+        @Override
+        public boolean isVirtual() {
+            return method instanceof HostedMethod ? ((HostedMethod) method).hasVTableIndex() : false;
+        }
+
+        @Override
+        public boolean isOverride() {
+            GraalError.unimplemented("Use MethodInfo.isOverride() instead.");
+            return false;
+        }
+
+        @Override
+        public int vtableOffset() {
+            /* TODO - convert index to offset (+ sizeof DynamicHub) */
+            return method instanceof HostedMethod ? ((HostedMethod) method).getVTableIndex() : -1;
```

Comment:
We also need to check if there is a vtable index.
```suggestion
            return isVirtual() ? ((HostedMethod) method).getVTableIndex() : -1;
```

Reply:
Done.